src/mlpack/tests/lrsdp_test.cpp

```diff
@@ -95,7 +95,8 @@ BOOST_AUTO_TEST_CASE(Johnson844LovaszThetaSDP)
 {
   // Load the edges.
   arma::mat edges;
-  data::Load("johnson8-4-4.csv", edges, true);
+  if (!data::Load("johnson8-4-4.csv", edges, true)) 
```

Comment:
```suggestion
  if (!data::Load("johnson8-4-4.csv", edges)) 
```
Third argument not required.